packs/bit-bucket/actions/create_issue.yaml

```diff
@@ -0,0 +1,26 @@
+name: create_issue
+runner_type: run-python
+description: Create an issue
+enabled: true
+entry_point: create_issue.py
+parameters:
+  repo:
+    type: string
+    description: Name of the repo
+    required: true
+  title:
+    type: string
+    description: Title of the issue to create
+    required: true
+  desc:
+    type: string
+    description: Description of the issue to create
+    required: true
+  status:
+    type: string
+    description: status of the issue, for example, bug, enahancement, proposal
+    required: true
+  kind:
```

Comment:
Looks like this could be an enum. See https://github.com/StackStorm/st2contrib/blob/master/packs/smartthings/actions/set_temperature.yaml#L8 for an example.